tests/Aspire.Hosting.Tests/ManifestGenerationTests.cs

```diff
@@ -81,6 +81,32 @@ public void EnsureWorkerProjectDoesNotGetBindingsGenerated()
         Assert.False(workerA.TryGetProperty("bindings", out _));
     }
 
+    [Fact]
+    public async Task WithContainerRegistryUpdatesContainerImageAnnotationsDuringPublish()
+    {
+        var builder = DistributedApplication.CreateBuilder(GetManifestArgs()).WithContainerRegistry("myprivateregistry.company.com");
+        var redis = builder.AddRedis("redis");
+        builder.Build().Run();
+
+        var redisManifest = await ManifestUtils.GetManifest(redis.Resource);
+        var expectedManifest = """
+            {
+              "type": "container.v0",
+              "connectionString": "{redis.bindings.tcp.host}:{redis.bindings.tcp.port}",
+              "image": "myprivateregistry.company.com/redis:7.2.4",
```

Comment:
NIT: I think that there was a recent change that made it such that you use a variable for the version instead of the version number so that it is easier to update the version in the future.

Reply:
This is what I was thinking about: https://github.com/dotnet/aspire/blob/1ee0f5acec3e5a8aea4e626e7b8002e9cdff4518/src/Aspire.Hosting.Redis/RedisContainerImageTags.cs#L9